src/test/java/io/github/hildi/can/service/StandardJavaSerializationServiceTest.java

```diff
@@ -0,0 +1,71 @@
+package io.github.hildi.can.service;
+
+import io.github.hildi.can.exceptions.DeserializeException;
+import io.github.hildi.can.exceptions.FileDoesNotExistsException;
+import io.github.hildi.can.exceptions.NotWritableFileException;
+import io.github.hildi.can.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.io.File;
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+
+/**
+ * Created by Serhii Hildi on 15.03.19.
+ */
+class StandardJavaSerializationServiceTest {
```

Comment:
let's divide this test into 2 different tests, one for deserialization and one for serialization